text.md

```diff
@@ -18,7 +18,7 @@ In general, you must give credit in such a way that the audience for your work c
 
 2.  this software project, by name, as a contribution to your work
 
-The audience for your work must be free to share, publish, and use that notice for any purpose.
+You must not prevent any member of the audience for your work from sharing, publishing, or using credits for any purpose.
```

Comment:
Not sure how I feel about this. It seems like we went from whitelist to blacklist, if that's even the correct term. 
1. Why should it be phrased in terms of forbidding restrictions, rather than required freedoms?
2. Why go from "the audience" to "any member of the audience"? What is the difference? (I think I'm ok with this change, but I'd still like to know).